mqtt-streaming/src/test/scala/docs/scaladsl/MqttSessionSpec.scala

```diff
@@ -200,6 +200,8 @@ class MqttSessionSpec
       client.offer(Command(disconnect))
 
       server.expectMsg(disconnectBytes)
+
+      result.futureValue shouldBe Done
```

Comment:
Prior to the kill switch change, the stream would not complete and this new assertion would fail. The kill switch ensures that the stream completes.